ignite/metrics/accumulation.py

```diff
@@ -69,6 +69,8 @@ def update(self, output: Union[float, torch.Tensor]) -> None:
             output = output.detach()
             if output.device != self._device:
                 output = output.to(self._device)
+            if output.dtype != self.accumulator.dtype:
+                output = output.to(self.accumulator.dtype)
```

Comment:
I think it would be better to make a single `.to` call instead of 2 in case when device and dtype are different.

Reply:
ok,  of course